docs/source/api/user-guide/polars/expressions/manipulation.ipynb

```diff
@@ -372,6 +372,38 @@
     "The output type is categorical, but with a data-independent encoding, meaning OpenDP allows grouping by these keys."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Drop NaNs\n",
+    "`.drop_nans` removes NaN float values from the data. Not to be confused with `.drop_nulls`. \n",
+    "While this proves that the data does not have NaNs, it also drops data invariants on dataset size.\n",
```

Comment:
I think I know what you mean, but "proves" feels odd. Could you say something like "This will guarantee that the downstream data will not have NaNs"?